Sources/MapboxCoreNavigation/Distance/RoadObjectDistanceInfo.swift

```diff
@@ -7,10 +7,29 @@ import MapboxNavigationNative
  * type/shape (gantry, polygon, line, point etc.).
  */
 public enum RoadObjectDistanceInfo {
+    /**
+     The information about distance to the road object represented as a point.
+     */
     case point(PointDistanceInfo)
```

Comment:
Once we inline these “info” types as associated values per https://github.com/mapbox/mapbox-navigation-ios/pull/2991#discussion_r631186845, those structs’ properties’ documentation comments will also be inlined here.